src/components/action-bar/action-bar.tsx

```
@@ -132,13 +154,16 @@
                 class={{
                     'is-full-width': this.layout === 'fullWidth',
                     'is-floating': this.layout === 'floating',
+                    'is-shrunk': this.actionBarIsShrunk && this.collapsible,
+                    'can-be-shrunk': !!this.collapsible,
                 }}
                 role="grid"
             >
                 <div class="items" role="rowgroup">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<tbody>
<tfoot>
<thead>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)